CyanogenMod 12

st7105 opened the pull request:
Why not?

dhiru1602 committed:
i9300: Audio updates

This is a combination of 3 commits.

audio: fix noise supression and update mixer ctls

Change-Id: I5b76be2d7d5d9754974295fce81b26498ccee391

audio: make builtin mic and back mic volumes consistent

Change-Id: I25cab57acb84a14200006e832a43676ff3c8b9d0

audio: set noise suppression based on audio parameter

Change-Id: Ie28205afda03280d9cc6437a7c98f7b995fd4f50

MarcKe committed:
i9300: Fix sensors & camera

dhiru1602 committed:
i9300: include: L bringup - update GPS header

Change-Id: I980a2fdb9c873e7942a1fb16cd0e3b42f8c834a8

dhiru1602 committed:
i9300: update mobile data interfaces

Change-Id: I0b8b34746d6d6ba633cba57bb7e49e510833ebeb

MarcKe committed:
Update fsbab for new syntax

MarcKe committed:
overlay: add config_deviceHardwareKeys again and set lte disabled

rickgaiser committed:
Add ambient display doze feature

Change-Id: I4e6fff788b51df1b51a443ca71b51883babf698b

MarcKe committed:
i9300: add notification/battery color overlays

Conflicts:
	overlay/frameworks/base/core/res/res/values/config.xml

JustArchi committed:
i9300: Update RIL header to V10

javifo committed:
i9300: Add RIL subclass for GSM

Fixes sim unlock with PIN code.

Change-Id: Idd6c0930743a3ba7e6834ea0b0e7d1175223c901

Conflicts:
	BoardConfig.mk

MarcKe committed:
i9300: enable multiusers

Commit:
i9300: Recorded Sound is distorted and crackly.
With value "18" instead "28" we have a good audio recording.

https://jira.cyanogenmod.org/browse/CYAN-6087

Change-Id: If96018c4b19761f35000d47ec2749b8d672cfe86

JustArchi committed:
i9300: Cleanup

JustArchi committed:
i9300: Use open-source camera again

JustArchi committed:
i9300: Overlay cleanup
This value was defined two times

JustArchi committed:
Create UPSTREAMS

JustArchi committed:
i9300: Fix compilation

JustArchi committed:
i9300: Temporarily disable SELinux

MarcKe committed:
i9300: disable broken keyboard workaround.

breaks keyboard completely on lollipop

MarcKe committed:
i9300: update hardwarekey overlay

config_deviceHardwareKeys=71 (home-1, back-2, menu-4, volume rockers-64)
config_deviceHardwareWakeKeys=65 (only home and volume rockers can wake it)

dhiru1602 committed:
i9300: init: cleanup

Change-Id: I76d330088c9fcf6e6ce20388980293d366af2997

MarcKe committed:
Revert "i9300: Audio updates"

This reverts commit a62c42123d733f5a5e2e1e3f89c6154054abc5bd.

MarcKe committed:
init: adjust lmk minfree values to kitkat ones

dhiru1602 committed:
i9300: audio: adjust microphone sensitivity

* Lower the main and sub mic gain to prevent distortion.
* Increase the headset mic gain
* Enable +30db boost for headset mic

Change-Id: If5a3605b8742fc125d36d2b11e9c2bb15f089f92

Conflicts:
	configs/tiny_hw.xml

cbalster committed:
i9300: fix sdcard fstab entries

mount point for vold managed volumes must be auto

Change-Id: I67c4cb82637ceb3d5b7c6392faec1770d5fad7a5

javifo committed:
sensors: fix light sensor

Change-Id: I7cad5c048c1a9877889216ca8ca43bb13daeb4b0

JustArchi committed:
fstab: Update external SD card entry

This will allow tools such as Link2SD work properly.
We don't need to emulate external sd card.

Commit:
Fix to no working pendrive :)

JustArchi committed:
Merge pull request #6 from DimitriMayakovsky/patch-1

Fix to no working pendrive :)

MarcKe committed:
i9300: init.target.rc: add vnswap configuration

JustArchi committed:
vnswap: Make it shiny
Just in case if we'd like to quad-swap sometime in the future

JustArchi committed:
vnswap: Handle kernels without vnswap

tonyshin12 commented:
So now I can just use thisto syNC and build cm12? 
I don't Hager to use justarchi's git?

MarcKe committed:
fstab: make it more readable

MarcKe committed:
Revert "Revert "i9300: Audio updates""

This reverts commit 1a8be87c537d15b61d50f32d6b7608e121be0f24.

MarcKe committed:
audio: fix incall mute

MarcKe committed:
audio: fix mute when switching devices

MarcKe committed:
i9300: disable dreams by default for the doze feature

MarcKe committed:
i9300: performance profiles added

Commit:
Enable wifi display

Commit:
Enable tethering

MarcKe committed:
Merge pull request #35 from DimitriMayakovsky/cm-12.0

Enable tethering

dhiru1602 committed:
audio: forward call mute to RIL client interface

Change-Id: Iccb763b48a687b88692c84976a7cddf72031b5be

MarcKe committed:
i9300: remove vnswap config

MarcKe committed:
zram: default size to 600MB over 4 devices

MarcKe committed:
i9300: swappiness set to 80

MarcKe committed:
Revert "i9300: swappiness set to 80"

This reverts commit 592eba3f53e81075541fedbf472b8a655fae33ab.

MarcKe committed:
Revert "zram: default size to 600MB over 4 devices"

This reverts commit 1e13faa2cb0150094ecd101ce29feadfa1ae607f.

MarcKe committed:
i9300: add zram configuration file

handling more than 1 zram device does not seem to be possible via fstab
so this is needed

MarcKe committed:
Revert "audio: fix mute when switching devices"

This reverts commit 755233519255224f511ce969e4d2b1ebed9423f4.

MarcKe committed:
Revert "audio: fix incall mute"

This reverts commit 5621533cb53755bc656d4a9aea097e84cd111f17.

MarcKe committed:
audio: wb_amr log

to be reverted later

MarcKe committed:
i9300: remove torch overlay

MarcKe committed:
i9300: enable sysfs torch handling

MarcKe committed:
i9300: audio: remove deprecated audio_stream_frame_size

MarcKe committed:
i9300: init.target.rc: overwrite file permissions of minfree and adj

MarcKe committed:
i9300: init.target.rc: overwrite ownership of minfree and adj

Commit:
Enable USB-OTG

IshmeetSKalra committed:
i9300: gps: update location based service server

Change-Id: Ia71a970c70b054b44f48dfb4af2db5b55c0d0d6c

JustArchi committed:
Merge branch 'cm-12.0' of https://github.com/ArchiDroid/android_device_samsung_i9300 into cm-12.0

JustArchi committed:
i9300: zRAM: Add priority support

Previously zram swaps were added with standard priorities: -1. -2, -3, -4. The data was not balanced between all of them, as zram0 had 50% more data than zram1 etc. Correct this to use all zram devices fairly, and detect if swapon supports priority (shouldn't be needed, but better safe than sorry).

Also simplify logic, we don't need to do math operations in such easy case.

MarcKe committed:
libsensors: add flags

MarcKe committed:
Revert "init: adjust lmk minfree values to kitkat ones"

This reverts commit 177431a2410d3d6c37c9c40423adccc82508bf9f.

MarcKe committed:
Revert "i9300: Temporarily disable SELinux"

This reverts commit 0d6cd60c07e92e5d220649a1ffd49f6dfd6ed2d0.

MarcKe committed:
i9300: selinux updates for cm12

this is a first draft, future changes will come

MarcKe committed:
i9300: selinux: further updates

MarcKe committed:
i9300: selinux: improve rules

fixes video recording and improves security

MarcKe committed:
i9300: selinux: updates for gps

this can most likely be done way more elegant

MarcKe committed:
i9300: selinux: small update to fix denial

MarcKe committed:
Revert "i9300: gps: update location based service server"

This reverts commit afff68381ba425a8e2c9d56c618ba3cc5fa3c0bb.

MarcKe committed:
i9300: selinux: small update for gpsd

MarcKe committed:
i9300: fix further selinux denial on gpsd

cm-gerrit commented:
Hi there;

This is an automated reply, to let you know CyanogenMod doesn't merge pull
requests submitted directly to our github.com repositories.

You can submit your change requests through our public review system,
located at http://review.cyanogenmod.org/

That review system is an installation of Google's Gerrit code review.
If you've never interacted with Gerrit before, we also suggest you take
a look at http://wiki.cyanogenmod.org/wiki/Howto:_Gerrit

Thanks!
